### DISCUSSION TITLE
A logo for iceberg rust!

### DISCUSSION THREAD

**Discussion by Xuanwo** [apache/iceberg-rust#516] [Ideas]
It must be cool for iceberg rust to have it's own logo! Maybe we can have a logo that a crab eating iceberg or the iceberg is rusty.

## Notes

Unlike [python's logo](https://www.python.org/community/logos/), the crab itself is not a logo owned by [rust foundation](https://foundation.rust-lang.org/policies/logo-policy-and-media-guide/). So it's fine for us to adding a cute crab in our logo.

**Comment by caldempsey** [comment]
I'd prefer a letter I with the RUST gears surrounding it ⚙️ , maybe frozen across a glacier as if it just got smacked with a superhero ice ability

**Comment by caldempsey** [comment]
![image](https://github.com/apache/iceberg-rust/assets/8885269/ea2a8b46-15fb-4aba-9a06-49c58bc0b821)

**Reply by liurenjie1024** [reply]
Hi, @caldempsey I'm about to raise a vote for iceberg-rust's logo, if the community decided to use this as our logo, would you donate the copyright of this picture to apache foundation?

**Reply by caldempsey** [reply]
yep

**Reply by liurenjie1024** [reply]
Thanks!

**Comment by liurenjie1024** [comment]
Looks cool!

**Comment by pmcgleenon** [comment]
![image](https://github.com/apache/iceberg-rust/assets/8458078/0e315b79-373f-412c-9117-e4cb6b00f7b1)

**Reply by liurenjie1024** [reply]
Hi, @pmcgleenon  I'm about to raise a vote for iceberg-rust's logo, if the community decided to use this as our logo, would you donate the copyright of this picture to apache foundation?

**Reply by RussellSpitzer** [reply]
Delicious Crab.

**Reply by pmcgleenon** [reply]
hi @liurenjie1024  a vote on the logo is a great idea!   I'm happy to donate the copyright if this one is selected

**Reply by liurenjie1024** [reply]
Thanks!

**Reply by flyrain** [reply]
I like this one better. Could we remove the snowflake in the crab and simplify the bottom part a bit?

**Reply by liurenjie1024** [reply]
> I like this one better. Could we remove the snowflake in the crab and simplify the bottom part a bit?

Yes, we will ask for help from designer to do the refactoring.

**Reply by SteveLauC** [reply]
I like this one, but can we tune the model a bit to make it more like a one made by human🤔
